images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
The busybox image used in cadvisor contains most of the necessary binaries. +1 for using that.